prox_config_backup.sh

```diff
@@ -62,16 +62,17 @@ trap clean_up EXIT
 
 # Don't change if not required
 _now=$(date +%Y-%m-%d.%H.%M.%S)
-_HOSTNAME=$(hostname -f)
+_HOSTNAME=$(hostname)
 _filename1="$_tdir/proxmoxetc.$_now.tar"
-_filename2="$_tdir/proxmoxpve.$_now.tar"
+_filename2="$_tdir/proxmoxvarlibpve.$_now.tar"
```

Comment:
I don't think it's a good idea to dump the whole /var/lib/pve dir. On systems that still use .qcow2 or other file based images, the backup would be huge.